### PR TITLE
Add Android issue template

### PR DESCRIPTION
**Pull request by srirambv**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Test Plan:
Fixes:  #7573

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by kjozwiak** [comment]
We should probably add  the following into the `Android` sheet now that we have rewards:

```
- Does the issue resolve itself when disabling Brave Rewards?
```

Quick question re: the following:

Screen Shot 2020-01-17 at 2 06 05 AM

Is that basically the `meta` for the template? When you're looking at the "fully formatted" document, you'll notice the following:

Screen Shot 2020-01-17 at 2 11 31 AM

Just want to make sure we're not going to be adding that into issues once this is merged 👍

File: .github/ISSUE_TEMPLATE/android-issue.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Device details
+   - Install type (ARM, x86):
+   - Device (Phone, Tablet, Phablet):
+   - Android version:
+   
+## Brave version
+
+
+### Website problems only
```
Comment (kjozwiak) [inline .github/ISSUE_TEMPLATE/android-issue.md]: We should probably add the following in this section as well:

```
- Does the issue resolve itself when disabling Brave Rewards?
```

Comment (srirambv) [reply .github/ISSUE_TEMPLATE/android-issue.md]: Updated

**Comment by srirambv** [comment]
@kjozwiak  This is what the metadata will show up when you select new issue
image
This is what is shown when you select either one just uses the meta data info and allows you to switch templates
image
This is what is shown after creating an issue 
image


This is already done on iOS repo. Basically doing the same thing as iOS repo does now.

**Comment by kjozwiak** [comment]
LGTM 👍

**Comment by srirambv** [comment]
@bsclifton  squash and merge isn't enabled so will just merge as it is

**Comment by kjozwiak** [comment]
Looks like both https://github.com/brave/brave-browser/issues/new?assignees=&labels=&template=android-issue.md&title= and https://github.com/brave/brave-browser/issues/new?assignees=&labels=&template=desktop.md&title= are working 👍 

Good job @srirambv!

**Pull request merged**